storage/tests/unit/test_bucket.py

```diff
@@ -245,6 +245,26 @@ def test_get_blob_hit(self):
         self.assertEqual(kw['method'], 'GET')
         self.assertEqual(kw['path'], '/b/%s/o/%s' % (NAME, BLOB_NAME))
 
+    def test_get_blob_hit_with_kwargs(self):
+        NAME = 'name'
+        BLOB_NAME = 'blob-name'
+        CHUNK_SIZE = 1024 * 1024
+        KEY = b'01234567890123456789012345678901'  # 32 bytes
+
+        connection = _Connection({'name': BLOB_NAME})
+        client = _Client(connection)
+        bucket = self._make_one(name=NAME)
+        blob = bucket.get_blob(
+            BLOB_NAME, client=client, encryption_key=KEY, chunk_size=CHUNK_SIZE
+        )
+        self.assertIs(blob.bucket, bucket)
+        self.assertEqual(blob.name, BLOB_NAME)
+        kw, = connection._requested
+        self.assertEqual(kw['method'], 'GET')
+        self.assertEqual(kw['path'], '/b/%s/o/%s' % (NAME, BLOB_NAME))
```

Comment:
The encryption headers are only local to the `get_blob` method so I don't know how I'd get access to them in the test.